packages/botonic-plugin-ner/src/model/ner.ts

```diff
@@ -2,38 +2,41 @@ import { Codifier } from '@botonic/nlp/dist/preprocess/codifier'
 import { Preprocessor } from '@botonic/nlp/dist/preprocess/preprocessor'
 import { Entity } from '@botonic/nlp/dist/tasks/ner/process/types'
 import { NerConfig } from '@botonic/nlp/dist/tasks/ner/storage/types'
-import { Locale } from '@botonic/nlp/dist/types'
 import { LayersModel } from '@tensorflow/tfjs'
 import { Tensor3D } from '@tensorflow/tfjs-core/dist/tensor'
+import { default as fetch } from 'node-fetch'
 
 import { PredictionProcessor } from '../postprocess/prediction-processor'
 import { InputGenerator } from '../preprocess/input-generator'
-import { getModelInfo } from '../utils/environment-utils'
+import { ModelInfoPromise } from '../types'
+
+// Support for fetch API in Node (Lambda Env): https://stackoverflow.com/a/48433898
+// @ts-ignore
+global.fetch = fetch
 
 export class NamedEntityRecognizer {
   model: LayersModel
   config: NerConfig
   inputGenerator: InputGenerator
   predictionProcessor: PredictionProcessor
 
-  constructor(readonly locale: Locale) {
+  constructor(modelInfoPromise: ModelInfoPromise) {
     // @ts-ignore
     return (async () => {
```

Comment:
@elozano98 @vanbasten17 
don't use async constructor. it's a hack (that's why you need to use ts-ignore). I haven't found a single web page recommending using it and that's why we don't use it in any TS project and why we avoided in botonic-nlp. Pages recommending not to use it and alternatives:
* https://rostacik.net/2017/04/23/how-to-handle-async-functions-inside-constructors-in-typescript/
* https://medium.com/@guillaume.viguierjust/dealing-with-asynchronous-constructors-in-typescript-c13c14c80954
* https://stackoverflow.com/a/47315507/145289

iife is also not recommended to use. See:
* https://github.com/thornjad/eslint-plugin-no-iife 
* https://levelup.gitconnected.com/why-its-time-to-stop-using-javascript-iifes-b62602f25bfc